news/_posts/core-weekly/2019-07-26-coreweekly-week-26-2019.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 26 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-07-26 15:30:00
+authors: [ AntoineThomas ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 24th to Sunday 30th of June 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+Core weekly edition is a bit late, mainly due to beginning of the summer and the first vacations of the team.
+
+So, how the 1.7.6.0 release? Tell us in the comments, now that you have a few experience with it.
```

Comment:
I think this will trigger negative answers 😐
I suggest we acknowledge that 1.7.6.0 contains major issues, like the currency bug, and confirm people that 1.7.6.1 is on the way.

Reply:
ok

Reply:
is it better now ?